css/style.css

```diff
@@ -1 +1,1091 @@
+  margin-bottom: 60px;
+}
+
+.faq-box details {
+  margin: 0 auto;
+  background-color: var(--secondary-color);
+  padding: 30px 20px;
+  width: 930px;
+  margin-bottom: 24px;
+}
+
+.faqs-name {
+  font-size: 24px;
+  line-height: 30px;
+  padding-bottom: 16px;
+  color: rgba(194, 195, 197, 1);
+  list-style: none;
+}
+
+.faqs-name::after {
+  content: url(../img/btn-icons/Plus.svg);
+  float: right;
+}
+
+.faqs-card[open] .faqs-name {
+  color: rgba(255, 255, 255, 1);
+}
+
+.faqs-card[open] .faqs-name::after {
+  content: url(../img/btn-icons/Minus.svg);
+}
+
+.faqs-content {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  border-top: 1px solid rgba(67, 67, 67, 1);
+  padding-top: 16px;
+}
+
+/* Footer */
+
+footer {
+  padding-top: 50px;
+}
+
+footer * {
+  background-color: transparent;
+}
+
+/*  call-action section */
+
+.call-action {
+  margin: 0 auto;
+  text-align: center;
+  margin-bottom: 80px;
+}
+
+.call-action p {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  margin-top: 8px;
+  margin-bottom: 40px;
+}
+
+.footer-btn-box {
+  margin-bottom: 40px;
+}
+
+.btn-footer {
+  background-color: rgba(88, 101, 242, 1);
+}
+
+/* social-networks Footer */
+
+.network-item {
+  margin: 0 auto;
+  background-color: rgba(194, 195, 197, 0.15);
+  width: 50px;
+  height: 50px;
+  margin-bottom: 20px;
+}
+
+.network-item img {
+  display: block;
+  margin: 0 auto;
+  padding: 14px 10px;
+  width: 26px;
+}
+
+/* footer-nav section */
+
+.footer-border {
+  border-top: 1px solid rgba(67, 67, 67, 1);
+  padding: 19px 0;
+}
+
+.footer-logo {
+  font-size: 22px;
+  line-height: 30px;
+  gap: 16px;
+}
+
+.footer-logo img {
+  width: 24px;
+  height: 26px;
+}
+
+.footer-logo-box p {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 14px;
+  line-height: 22px;
+  text-align: right;
+}
+
+/* Footer Pages */
+
+.nav-pages-item a {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 14px;
+  line-height: 22px;
+  text-align: right;
+}
+
+/************************* Catalog Page **************************/
+
+/* Breadcrumbs */
+
+.breadcrumbs {
+  background-color: var(--secondary-color);
+  padding-top: 77px;
+  padding-bottom: 77px;
+  text-align: center;
+}
+
+.breadcrumbs * {
+  background-color: transparent;
+}
+
+.breadcrumbs ul {
+  margin-top: 12px;
+  display: flex;
+  justify-content: center;
+}
+
+.breadcrumbs a {
+  padding: 0 8px;
+}
+
+.breadcrumbs a,
+.breadcrumbs span {
+  font-size: 14px;
+  line-height: 20px;
+  justify-content: center;
+}
+
+.inactive-link a {
+  color: var(--inactive-color);
+  border-right: 2px solid var(--inactive-color);
+}
+
+/* NFT Collection */
+
+.nft-collection {
+  background-color: var(--main-background-color);
+  padding-top: 100px;
+  padding-bottom: 200px;
+}
+
+.nft-collection h2 {
+  text-align: left;
+  margin-bottom: 40px;
+  background-color: transparent;
+}
+
+/* Call Action section - coloful */
+
+.CTA-section {
```

Comment:
Avoid using uppercase letters in class names:
Class names in HTML and CSS are case-sensitive, and using lowercase letters is a standard convention that helps avoid potential errors and inconsistencies.

Reply:
#fixed